vazhin/app.js

```diff
@@ -0,0 +1,283 @@
+// Selectors
+
+const todayDate = document.querySelector(".today-date");
+const todoList = document.querySelector(".todo-list");
+const taskText = document.querySelector('#todo-text');
+const priorityInput = document.querySelector('#select-priority');
+const deadlineInput = document.querySelector('#date');
+const addBtn = document.querySelector(".btn");
+const cancelBtn = document.querySelector(".cancel-btn");
+const noTasks = document.querySelector(".tasks-empty");
+let arrOfTasks = [];
+let inEditMode = false;
+let indexOfInEditTask;
+priorityInput.value = "";
+
+// Event listeners
+
+window.addEventListener('load', startDateTime);
+addBtn.addEventListener("click", addTodo);
+cancelBtn.addEventListener("click", cancelTodo);
+
+// Functions
+
+function addTodo(event) {
+    event.preventDefault();
+
+    if (taskText.value === "") {
+
+        taskText.classList.add("fill-out-field");
+        taskText.placeholder = "add a task";
+        return;
```

Comment:
a better way to do this is before returning, check all the inputs and validate them, then give feedback to the user (a 3 red-bordered input if all of them are empty rather than 1 by 1)